doc/themes/the-things-stack/layouts/partials/file-info.html

```diff
@@ -1,18 +1,5 @@
 <div class="content">
-<p class="has-text-light is-size-7">
-  Last changed{{ with .GitInfo }} by {{ .AuthorName }}{{ end }} on {{ .Lastmod.Format "02 Jan 2006" }}.
-  <br>
-  {{ if .GitInfo -}}
-  {{- if .Site.Params.github_repository -}}
-    <a href="{{ .Site.Params.github_repository }}/commit/{{ .GitInfo.Hash }}">
-      {{ .GitInfo.Subject }}
-    </a>
-  {{- else -}}
-    {{ .GitInfo.Subject }}
-  {{- end }}{{ end }}
+<p class="has-text-light is-size-2">
```

Comment:
I think we should put the version somewhere in the top of the page (where people can see it), and not in the footer (where people don't see it unless they go looking for it)